src/json-formatter.html

```diff
@@ -9,9 +9,12 @@
       </span>
       <span ng-if="!isObject()" ng-click="openLink(isUrl)" class="{{type}}" ng-class="{date: isDate, url: isUrl}">{{parseValue(json)}}</span>
     </span>
+    <span ng-if="showThumbnail() && isObject() && !isOpen" class="thumbnail-text">
```

Comment:
preview may be Number or String or ...
for “value” type‘s field，preview of little significance
I chose here to hide the non-object and array types Preview